README.md

```diff
@@ -71,300 +55,84 @@ overhead compared to a Java VM. For example this allows to create very
-[Guide how to use Appkit from Python by bridging with JPype](https://github.com/ergoplatform/ergo-appkit/wiki/Using-Appkit-from-Python)) 
+#### Maven
 
-## Repository organization
+Appkit is **Java/Scala** library which is cross compiled by both Scala 2.11 and Scala 2.12
+compilers. As a result it can run on JVMv7/[Android](https://github.com/aslesarenko/ergo-android) 
+(when it is built with Scala 2.11) and JVMv8 and above (when it is built with Scala 2.12).
 
-| sub-module  | description |
-|---|-----|
-| common  |  Used in all other submodules and contain basic utility classes |
-| java-client-generated  | Typed REST Java client [generated](https://swagger.io/tools/swagger-codegen/) from [Swagger definition](https://swagger.io/specification/) of [Ergo node API](https://github.com/ergoplatform/ergo/blob/master/src/main/resources/api/openapi.yaml) |
-| lib-api   | All Appkit Java interfaces which can be implemented elsewhere |
-| lib-impl  | Implementation of Appkit interfaces using `java-client-generated` to connect to Ergo API  |
-| examples | collection of simple Appkit example applications   |
+Appkit is released on [Maven Central](https://mvnrepository.com/artifact/org.ergoplatform/ergo-appkit) 
+and you can add it to your project with SBT, Maven or Gradle:
 
-## Setup
+Maven: 
 
-### Prerequisites
-
-Appkit is Java/Scala library which is cross compiled by both Scala 2.11 and Scala 2.12
-compilers. As a result it can run on JVMv7 (when it is built with Scala 2.11) and JVMv8
-and above (when it is built with Scala 2.12). In addition Appkit jar built with Scala 2.11
-can be [used as a library in Android
-application](https://github.com/aslesarenko/ergo-android).
+    <dependency>
+        <groupId>org.ergoplatform</groupId>
+        <artifactId>ergo-appkit_2.12</artifactId>
+        <version>4.0.9</version>
+    </dependency>
 
-For using Appkit from non-JVM languages (JS, Python, Ruby) GraalVM is required (Community
-or Enterprise edition). It can be [downloaded](https://www.graalvm.org/downloads/) and
-installed on the computer where the App is supposed to run. Community
-edition should be enough for Ergo Appkit library.
+Gradle:
 
-#### Install GraalVM Community Edition on MacOS
+     implementation 'org.ergoplatform:ergo-appkit_2.12:4.0.9'
 
-First you need to download an archive with the [latest
-release](https://github.com/oracle/graal/releases) of GraalVM (e.g.
-`graalvm-ce-darwin-amd64-19.2.1.tar.gz` at the time of writing) for MacOS and
-put the programs from it onto the `$PATH`.
+(other examples on the Maven Central page).
 
-```shell
-$ cd <your/directory/with/downloaded/graal>
-$ tar -zxf graalvm-ce-darwin-amd64-19.2.1.tar.gz
-$ export GRAAL_HOME=<your/directory/with/downloaded/graal>/graalvm-ce-19.2.1/Contents/Home
-$ export PATH=$PATH:${GRAAL_HOME}/bin
-```
+As an alternative, you can download the jar file from the 
+[releases page](https://github.com/ergoplatform/ergo-appkit/releases) and add it to your project manually.
```

Comment:
Jar files are not created for releases.

Reply:
Oh yes they are! https://github.com/ergoplatform/ergo-appkit/blob/315c6d36955756e0f78da63e214a12d4221568b2/.github/workflows/release.yml#L46 - stored on the action, I copy them manually to the releases page each time. Could be done from the action as well.

Reply:
I looked at the releases, no jars, so maybe first setup auto-attachment of jars?

Reply:
[This release](https://github.com/ergoplatform/ergo-appkit/releases/tag/v4.0.9) has the jar attached, but I can add a change to automatically add it from the action here as well.